gradle.properties

```diff
@@ -1,5 +1,4 @@
-baseApiVersion=1.0.0-SNAPSHOT
-geyserApiVersion=1.0.1-SNAPSHOT
+baseApiVersion=1.0.1-SNAPSHOT
```

Comment:
Should be 1.1.0 because new functionality has been added, and other functionality has been deprecated.

Reply:
Resolving this as geyver allows for non-breaking deprecations during a minor bump